Anature/src/application/anatures/moves/Move.java

```diff
@@ -140,38 +157,69 @@ public int getTotalMovePoints()
 		return mTotalMovePoints;
 	}
 
+	public int getMovePower()
+	{
+		return mMovePower;
+	}
+
 	public double getAccuracy()
 	{
 		return mAccuracy;
 	}
 
+	/*
+	 * PROTECTED METHODS
+	 */
+
+	protected int calculateDamage(IAnature source, IAnature target, boolean isSpecialMove)
+	{
+		IStats sourceStats = source.getStats();
+		IStats targetStats = target.getStats();
+
+		double levelCalculation = ((2.0 * (double) sourceStats.getLevel()) / 5.0) + 2.0;
+		double attackStatCalculation = isSpecialMove ? sourceStats.getTotalSpecialAttack() : sourceStats.getTotalAttack();
+		double movePowerCalculation = isSpecialMove ? (double) getMovePower() / (double) targetStats.getTotalSpecialDefense() : (double) getMovePower() / (double) targetStats.getTotalDefense();
```

Comment:
While this technically does the same thing as the pokemon formula, you might want to change it to match the formula.
In pokemon's damage formula, it does "Power * (Attack / Defense)" so its technically the same thing, but makes more sense to read out aloud than "Attack * (Power / Defense)". That's just my opinion though.

Reply:
Ok I rearranged the variables to be in that order